api/src/main/java/org/openmrs/module/appointments/notification/impl/DefaultTCAppointmentPatientEmailNotifier.java

```diff
@@ -68,14 +73,41 @@ public NotificationResult sendNotification(final Appointment appointment) throws
             try {
                 log.info("Sending mail through: " +  mailSender.getClass());
                 mailSender.send(emailSubject, emailBody, new String[] { patientEmail }, null, null);
-                return new NotificationResult("", "EMAIL", 0, EMAIL_SENT);
+                return new NotificationResult("", "EMAIL", NotificationResult.SUCCESS_STATUS, EMAIL_SENT);
             } catch (Exception e) {
                 log.error(EMAIL_FAILURE, e);
                 throw new NotificationException(EMAIL_FAILURE, e);
             }
         } else {
             log.warn(EMAIL_NOT_CONFIGURED);
-            return new NotificationResult(null, "EMAIL", 1, EMAIL_NOT_CONFIGURED);
+            return new NotificationResult(null, "EMAIL", NotificationResult.GENERAL_ERROR, EMAIL_NOT_CONFIGURED);
+        }
+    }
+
+    public NotificationResult sendNotification(final Patient patient, final String provider, final String link) throws NotificationException {
```

Comment:
I think this method is redundant .. we can use the same "sendNotiication(appointment)"

Reply:
@angshu As explained in the above comment it would be better to maintain separate methods as it might complicate the basic appointment flow

Reply:
@angshu: Shall we mark this conversation as resolved?

Reply:
See comment above